src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -84,6 +88,8 @@ object HttpGoogleServicesDAO extends GoogleServicesDAO with FireCloudRequestBuil
   val authScopes = Seq("profile", "email")
   // the minimal scope to read from GCS
   val storageReadOnly = Seq(StorageScopes.DEVSTORAGE_READ_ONLY)
+  // scope for creating anonymized Google groups
+  val directoryScope = Seq("https://www.googleapis.com/auth/admin.directory.group")
```

Comment:
This string is also defined in `DirectoryScopes.ADMIN_DIRECTORY_GROUP`. I think this should also work and more more ideomatic:
```suggestion
  val directoryScope = Seq(DirectoryScopes.ADMIN_DIRECTORY_GROUP)
```